src/WebJobs.Extensions.DurableTask/LinuxLoggerProvider.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class LinuxLoggerProvider : ILoggerProvider
```

Comment:
Same comments about prefixing with "AppService" and adding copyright headers at the top of each new file.